### PR TITLE
Add a new team member

### PR DESCRIPTION
**Pull request by ania-hm**

### CODE REVIEW

File: db/seeds/07_contributions.rb [hunk 1]
```diff
@@ -3,3 +3,4 @@
 require_relative 'contributions/lisu'
 require_relative 'contributions/torrocus'
 require_relative 'contributions/womanonrails'
+require_relative 'contributions/ania'
```
Comment (torrocus) [inline db/seeds/07_contributions.rb:6]: You are the first, not the last. :)

File: db/seeds/contributions/ania.rb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+# ania in blog
+params = { person_id: ania.id, project_id: blog.id }
+
+contribution = Contribution.new(params.merge(position: 1))
+contribution.attributes = { locale: :en, name: 'Copywriting' }
+contribution.attributes = { locale: :pl, name: 'Pisanie artykułów' }
+contribution.save
+
+contribution = Contribution.new(params.merge(position: 2))
+contribution.attributes = { locale: :en, name: 'Typo correction' }
```
Comment (torrocus) [inline db/seeds/contributions/ania.rb:18]:
```suggestion
contribution.attributes = { locale: :en, name: 'Typos correction' }
```

File: db/seeds/team/ania.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  rspec
+  minitest
+  playwright
+  ruby
+  ruby-on-rails
+  jekyll
+  sass
+  slim
+  sonic-pi
+].each do |slug|
```
Comment (torrocus) [inline db/seeds/team/ania.rb:46]: +svg

File: db/seeds/team/ania.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
+ania = Person.where(nickname: 'ania').first_or_initialize
+ania.assign_attributes(
+  description: 'Programming adept',
+  fullname: 'Anna Bargieł',
+  github: 'ania-hm',
+  image: 'team/test-image.jpg',
```
Comment (torrocus) [inline db/seeds/team/ania.rb:6]: ania.jpg

Comment (ania-hm) [reply db/seeds/team/ania.rb:6]: failure ania.jpg

**Review by torrocus — APPROVED** [review]

**Pull request merged**